diff --git a/bun-sql/driver.js b/bun-sql/driver.js
index 8201c50ae44cc5ff206e425d2656f9a5678de69b..c9dff5ea7469feaed487b14af7839385a89d49c1 100644
--- a/bun-sql/driver.js
+++ b/bun-sql/driver.js
@@ -1,4 +1,3 @@
-import { SQL } from "bun";
 import { entityKind } from "../entity.js";
 import { DefaultLogger } from "../logger.js";
 import { PgDatabase } from "../pg-core/db.js";
@@ -43,7 +42,7 @@ function construct(client, config = {}) {
 }
 function drizzle(...params) {
   if (typeof params[0] === "string") {
-    const instance = new SQL(params[0]);
+    const instance = new Bun.SQL(params[0]);
     return construct(instance, params[1]);
   }
   if (isConfig(params[0])) {
@@ -51,10 +50,10 @@ function drizzle(...params) {
     if (client) return construct(client, drizzleConfig);
     if (typeof connection === "object" && connection.url !== void 0) {
       const { url, ...config } = connection;
-      const instance2 = new SQL({ url, ...config });
+      const instance2 = new Bun.SQL({ url, ...config });
       return construct(instance2, drizzleConfig);
     }
-    const instance = new SQL(connection);
+    const instance = new Bun.SQL(connection);
     return construct(instance, drizzleConfig);
   }
   return construct(params[0], params[1]);
